spec/features/filter_peeps_spec.rb

```diff
@@ -0,0 +1,15 @@
+feature 'Filtering peeps' do
+  scenario 'user can filter peeps by keyword' do
+    setup_test_database
+
+    Peeps.create(message: 'Test peep!', entry_date: "2021-03-18")
+    Peeps.create(message: 'Second test peep!', entry_date: "2021-02-15")
+
+    visit ('/peeps')
+    fill_in('filter', with: 'peep')
```

Comment:
Given that both test peeps contain the word 'peep' maybe another word could have been used to test the filter function, such as 'Second'? Even better would be 'second' as then you test your downcase code as well.

Reply:
Good points!